src/burgerProgramm/Gemuese.java

```diff
@@ -17,11 +17,10 @@ public double berechneHoehe() {
 
 	public String zubereiten() {
 		String temp = this.name + " wird gewaschen.\n";
-
-		for (int i = 1; i <= this.scheibenAnzahl - 1; i++) {
+	
+		for (int i = 1; i <= this.scheibenAnzahl; i++) {
 			temp += "     " + i + ". Scheibe mit " + this.scheibenDicke + " mm schneiden.\n";
 		}
-		temp += "     " + this.scheibenAnzahl + ". Scheibe mit " + this.scheibenDicke + " mm schneiden.";
 		return temp;
```

Comment:
Zeile 24 darf nicht entfernt werden, da sonst eine lücke im ausgabe text ist wegen \n

--> in der for-schleife eine if-bedingung einfügen, die das \n zu temp hinzufügt, solange i nicht scheibenAnzahl entspricht.

in 23 einfügen->  if (!(i == this.scheibenAnzahl)){temp += "\n";}
-> \n aus zeile 22 entfernen